prow/plugins/approve/approve.go

```diff
@@ -462,7 +463,17 @@ func handle(log *logrus.Entry, ghc githubClient, repo approvers.Repo, githubConf
 	start = time.Now()
 	notifications := filterComments(commentsFromIssueComments, notificationMatcher(botUserChecker))
 	latestNotification := getLast(notifications)
-	newMessage := updateNotification(githubConfig.LinkURL, opts.CommandHelpLink, opts.PrProcessLink, pr.org, pr.repo, pr.branch, latestNotification, approversHandler)
+
+	var testFreezeResult *testfreeze.Result
+	// Enable test freeze only for the k/k repo and master branch
+	if pr.org == "kubernetes" && pr.repo == "kubernetes" && pr.branch == "master" {
+		testFreezeResult, err = testfreeze.NewChecker(log).InTestFreeze()
+		if err != nil {
+			return fetchErr("Test Freeze status", err)
```

Comment:
Would it makes sense to "survive" this error and just log it and continue? The point of the change is to add additional information to the comment, it probably does not make sense to abort the rest of the handler on e.g. a random git flake.

Reply:
Yes, we now only log the error if we're unable to fetch the status result.